vignettes/methodological_introduction.Rmd

```diff
@@ -0,0 +1,109 @@
+In a clinical trial, these time points are typically visits according to a schedule that is pre-defined in the trial protocol.
+The distinguishing feature of MMRMs, compared to other implementations of linear mixed models, is that subject-specific random effects (which are not of direct interest for estimation and inference) are considered as residual effects, i.e. they are part of the error correlation matrix.
+This vignette provides a brief methodological introduction to MMRMs.
+MMRMs are described as an extension to a basic linear mixed-effects model.
+The aim is to provide a basic orientation and guide for applied statisticians working in the field of clinical trials regarding theoretical underpinnings of MMRMs. In the descriptions of the models below, we generally follow @Pinheiro2000 and @Mallinckrodt2017. 
+
+
+## The basic linear mixed-effects model
+
+@Laird1982 introduced the basic linear mixed-effects model for a single level of grouping to be any model that expresses an $n_i$-dimensional response (column) vector $y_i$ for the $i$th subject (or, more generally, group or unit) as
+
+\[
+ y_i = X_i \beta + Z_i b_i + \epsilon_i, \quad i=1,\ldots,M ,\\
+ b_i \sim \mathcal{N}(0,\Psi), \quad \epsilon_i \sim \mathcal{N}(0,\sigma^2I),
+\]
+where 
+$\beta$ is the $p$-dimensional vector of fixed effects,
+$b$ is the $q$-dimensional vector of random patient-specific effects, 
+$X_i$ (of size $n_i \times p$) and $Z_i$ (of size $n_i \times q$) are known regressor matrices relating observations to the fixed-effects and random-effects, respectively, and 
+$\epsilon_i$ is the $n_i$-dimensional within-subject error.
+The random effects $b_i$ and the within-group errors $\epsilon_i$ are assumed to  follow a normal distribution, with means of $0$ and variance-covariance matrices  $\Psi$ and $\sigma^2I$, where $I$ is an identity matrix. They are assumed to be independent for different subjects and independent of each other for the same subject.
+The random effects $b_i$ describe the shift from the mean of the linear predictor for each subject. As they are defined to have a mean of $0$, any
+non-zero mean for a term in the random effects must be expressed as part
+of the fixed-effects terms. Therefore, the columns of $Z_i$ are usually a (small) subset of the columns of $X_i$.
+
+Mixed models are called ''mixed'' because they consider both fixed and random effects and thus allow considerable modeling flexibility. This is the case even for the basic formulation described above. However, it still restricts within-group errors to be independent, identically distributed random variables with mean of zero and constant variance. These assumptions may often be seen as too restrictive (unrealistic) for applications. For example, in the case of clinical trials with  repeated measurements of subjects over time, observations are not independent and within-subject correlation needs to be accounted for by the model.
```

Comment:
```suggestion
Mixed models are called "mixed" because they consider both fixed and random effects and thus allow considerable modeling flexibility. This is the case even for the basic formulation described above. However, it still restricts within-group errors to be independent, identically distributed random variables with mean of zero and constant variance. These assumptions may often be seen as too restrictive (unrealistic) for applications. For example, in the case of clinical trials with  repeated measurements of subjects over time, observations are not independent and within-subject correlation needs to be accounted for by the model.
```